test/unit/calculators/uk_visa_calculator_test.rb

```diff
@@ -391,6 +391,23 @@ class UkVisaCalculatorTest < ActiveSupport::TestCase
         end
       end
 
+      context "#passport_country_is_hong_kong_british_national_overseas?" do
+        should 'return true if passport_country is "hong-kong-(british-national-overseas"' do
+          calculator = UkVisaCalculator.new
+          calculator.passport_country = "hong-kong-(british-national-overseas)"
+          assert calculator.passport_country_is_hong_kong_british_national_overseas?
+        end
+
+        should 'return false if passport_country is not "hong-kong-(british-national-overseas"' do
+          calculator = UkVisaCalculator.new
+          calculator.passport_country = "not-hong-kong-(british-national-overseas)"
+          assert_not calculator.passport_country_is_hong_kong_british_national_overseas?
```

Comment:
The assertion above seems a bit redundant, as you're testing it in the assertion below. I prefer the real world example so suggest deleting the one above.

Reply:
Thanks @ChrisBAshton, sensible suggestion. This is going for content-review and changes are expected, so I will leave this as is for now but make the change when further work on the branch is needed.

Reply:
Following content and department review the requirements have changed so this test and method no longer exist :)